WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/BlockListItem.kt

```diff
@@ -74,6 +78,12 @@ sealed class BlockListItem(val type: Type) {
         val menuAction: ((View) -> Unit)? = null
     ) : BlockListItem(TITLE)
 
+    data class TitleWithMore(
+        @StringRes val textResource: Int? = null,
+        val text: String? = null,
+        val menuAction: ((View) -> Unit)? = null
```

Comment:
Would it be better to rename `menuAction` -> `moreAction`? There was a menu in the previous design, but now there is a more button.